[CI] ClusterStateTests testGetMinTransportVersion failing

The failure is from a branch I have been working on, but I repeatedly run it on main and it failed only with this seed.

**Build scan:**
https://gradle-enterprise.elastic.co/s/mo4iivn33cx3k/tests/:server:test/org.elasticsearch.cluster.ClusterStateTests/testGetMinTransportVersion

**Reproduction line:**
```
./gradlew ':server:test' --tests "org.elasticsearch.cluster.ClusterStateTests.testGetMinTransportVersion" -Dtests.seed=8476C65EED3EADC6 -Dtests.locale=en-IN -Dtests.timezone=US/East-Indiana -Druntime.java=20
```

**Applicable branches:**
main

**Reproduces locally?:**
Yes

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.cluster.ClusterStateTests&tests.test=testGetMinTransportVersion

**Failure excerpt:**
```
java.lang.AssertionError: 
Expected: <8060299>
     but: was <8000099>

  at __randomizedtesting.SeedInfo.seed([8476C65EED3EADC6:120A66DF9D21AF6D]:0)
  at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
  at org.junit.Assert.assertThat(Assert.java:956)
  at org.junit.Assert.assertThat(Assert.java:923)
  at org.elasticsearch.cluster.ClusterStateTests.testGetMinTransportVersion(ClusterStateTests.java:1107)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1623)

```

--- a/server/src/test/java/org/elasticsearch/cluster/ClusterStateTests.java
+++ b/server/src/test/java/org/elasticsearch/cluster/ClusterStateTests.java
@@ -214,7 +214,7 @@ public class ClusterStateTests extends ESTestCase {
                   "transport_versions" : [
                     {
                       "node_id" : "nodeId1",
-                      "transport_version" : "8000099"
+                      "transport_version" : "%s"
                     }
                   ],
                   "metadata": {
@@ -364,7 +364,7 @@ public class ClusterStateTests extends ESTestCase {
                       "nodeId1": []
                     }
                   }
-                }""", ephemeralId, Version.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId)),
+                }""", ephemeralId, Version.CURRENT, TransportVersion.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId)),
             Strings.toString(builder)
         );
 
@@ -455,7 +455,7 @@ public class ClusterStateTests extends ESTestCase {
                   "transport_versions" : [
                     {
                       "node_id" : "nodeId1",
-                      "transport_version" : "8000099"
+                      "transport_version" : "%s"
                     }
                   ],
                   "metadata" : {
@@ -601,7 +601,7 @@ public class ClusterStateTests extends ESTestCase {
                       "nodeId1" : [ ]
                     }
                   }
-                }""", ephemeralId, Version.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId),
+                }""", ephemeralId, Version.CURRENT, TransportVersion.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId),
             Strings.toString(builder)
         );
 
@@ -693,7 +693,7 @@ public class ClusterStateTests extends ESTestCase {
                   "transport_versions" : [
                     {
                       "node_id" : "nodeId1",
-                      "transport_version" : "8000099"
+                      "transport_version" : "%s"
                     }
                   ],
                   "metadata" : {
@@ -845,7 +845,7 @@ public class ClusterStateTests extends ESTestCase {
                       "nodeId1" : [ ]
                     }
                   }
-                }""", ephemeralId, Version.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId),
+                }""", ephemeralId, Version.CURRENT, TransportVersion.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId),
             Strings.toString(builder)
         );
 

--- a/server/src/test/java/org/elasticsearch/cluster/ClusterStateTests.java
+++ b/server/src/test/java/org/elasticsearch/cluster/ClusterStateTests.java
@@ -995,7 +995,7 @@ public class ClusterStateTests extends ESTestCase {
                     .add(new DiscoveryNode("nodeId1", new TransportAddress(InetAddress.getByName("127.0.0.1"), 111), Version.CURRENT))
                     .build()
             )
-            .putTransportVersion("nodeId1", TransportVersion.V_8_0_0)
+            .putTransportVersion("nodeId1", TransportVersion.CURRENT)
             .blocks(
                 ClusterBlocks.builder()
                     .addGlobalBlock(

--- a/server/src/test/java/org/elasticsearch/cluster/ClusterStateTests.java
+++ b/server/src/test/java/org/elasticsearch/cluster/ClusterStateTests.java
@@ -150,221 +150,232 @@ public class ClusterStateTests extends ESTestCase {
         builder.endObject();
 
         assertEquals(
-            XContentHelper.stripWhitespace(Strings.format("""
-                {
-                  "cluster_uuid": "clusterUUID",
-                  "version": 0,
-                  "state_uuid": "stateUUID",
-                  "master_node": "nodeId1",
-                  "blocks": {
-                    "global": {
-                      "1": {
-                        "description": "description",
-                        "retryable": true,
-                        "disable_state_persistence": true,
-                        "levels": [
-                          "read",
-                          "write",
-                          "metadata_read",
-                          "metadata_write"
-                        ]
-                      }
-                    },
-                    "indices": {
-                      "index": {
-                        "2": {
-                          "description": "description2",
-                          "retryable": false,
-                          "levels": [
-                            "read",
-                            "write",
-                            "metadata_read",
-                            "metadata_write"
-                          ]
-                        }
-                      }
-                    }
-                  },
-                  "nodes": {
-                    "nodeId1": {
-                      "name": "",
-                      "ephemeral_id": "%s",
-                      "transport_address": "127.0.0.1:111",
-                      "external_id": "",
-                      "attributes": {},
-                      "roles": [
-                        "data",
-                        "data_cold",
-                        "data_content",
-                        "data_frozen",
-                        "data_hot",
-                        "data_warm",
-                        "index",
-                        "ingest",
-                        "master",
-                        "ml",
-                        "remote_cluster_client",
-                        "search",
-                        "transform",
-                        "voting_only"
-                      ],
-                      "version": "%s"
-                    }
-                  },
-                  "transport_versions" : [
-                    {
-                      "node_id" : "nodeId1",
-                      "transport_version" : "%s"
-                    }
-                  ],
-                  "metadata": {
-                    "cluster_uuid": "clusterUUID",
-                    "cluster_uuid_committed": false,
-                    "cluster_coordination": {
-                      "term": 1,
-                      "last_committed_config": [
-                        "commitedConfigurationNodeId"
-                      ],
-                      "last_accepted_config": [
-                        "acceptedConfigurationNodeId"
-                      ],
-                      "voting_config_exclusions": [
+            XContentHelper.stripWhitespace(
+                Strings.format(
+                    """
                         {
-                          "node_id": "exlucdedNodeId",
-                          "node_name": "excludedNodeName"
-                        }
-                      ]
-                    },
-                    "templates": {
-                      "template": {
-                        "order": 0,
-                        "index_patterns": [
-                          "pattern1",
-                          "pattern2"
-                        ],
-                        "settings": {
-                          "index": {
-                            "version": {
-                              "created": "%s"
-                            }
-                          }
-                        },
-                        "mappings": {
-                          "key1": {}
-                        },
-                        "aliases": {}
-                      }
-                    },
-                    "indices": {
-                      "index": {
-                        "version": 1,
-                        "mapping_version": 1,
-                        "settings_version": 1,
-                        "aliases_version": 1,
-                        "routing_num_shards": 1,
-                        "state": "open",
-                        "settings": {
-                          "index": {
-                            "number_of_shards": "1",
-                            "number_of_replicas": "2",
-                            "version": {
-                              "created": "%s"
-                            }
-                          }
-                        },
-                        "mappings": {
-                          "type": {
-                            "type1": {
-                              "key": "value"
-                            }
-                          }
-                        },
-                        "aliases": [
-                          "alias"
-                        ],
-                        "primary_terms": {
-                          "0": 1
-                        },
-                        "in_sync_allocations": {
-                          "0": [
-                            "allocationId"
-                          ]
-                        },
-                        "rollover_info": {
-                          "rolloveAlias": {
-                            "met_conditions": {},
-                            "time": 1
-                          }
-                        },
-                        "system": false,
-                        "timestamp_range": {
-                          "shards": []
-                        },
-                        "stats": {
-                            "write_load": {
-                              "loads": [-1.0],
-                              "uptimes": [-1]
+                          "cluster_uuid": "clusterUUID",
+                          "version": 0,
+                          "state_uuid": "stateUUID",
+                          "master_node": "nodeId1",
+                          "blocks": {
+                            "global": {
+                              "1": {
+                                "description": "description",
+                                "retryable": true,
+                                "disable_state_persistence": true,
+                                "levels": [
+                                  "read",
+                                  "write",
+                                  "metadata_read",
+                                  "metadata_write"
+                                ]
+                              }
                             },
-                            "avg_size": {
-                                "total_size_in_bytes": 120,
-                                "shard_count": 1
-                            }
-                        },
-                        "write_load_forecast" : 8.0
-                      }
-                    },
-                    "index-graveyard": {
-                      "tombstones": []
-                    },
-                    "reserved_state" : { }
-                  },
-                  "routing_table": {
-                    "indices": {
-                      "index": {
-                        "shards": {
-                          "0": [
-                            {
-                              "state": "STARTED",
-                              "primary": true,
-                              "node": "nodeId2",
-                              "relocating_node": null,
-                              "shard": 0,
-                              "index": "index",
-                              "allocation_id": {
-                                "id": "%s"
-                              },
-                              "relocation_failure_info" : {
-                                "failed_attempts" : 0
+                            "indices": {
+                              "index": {
+                                "2": {
+                                  "description": "description2",
+                                  "retryable": false,
+                                  "levels": [
+                                    "read",
+                                    "write",
+                                    "metadata_read",
+                                    "metadata_write"
+                                  ]
+                                }
                               }
                             }
-                          ]
-                        }
-                      }
-                    }
-                  },
-                  "routing_nodes": {
-                    "unassigned": [],
-                    "nodes": {
-                      "nodeId2": [
-                        {
-                          "state": "STARTED",
-                          "primary": true,
-                          "node": "nodeId2",
-                          "relocating_node": null,
-                          "shard": 0,
-                          "index": "index",
-                          "allocation_id": {
-                            "id": "%s"
                           },
-                          "relocation_failure_info" : {
-                            "failed_attempts" : 0
+                          "nodes": {
+                            "nodeId1": {
+                              "name": "",
+                              "ephemeral_id": "%s",
+                              "transport_address": "127.0.0.1:111",
+                              "external_id": "",
+                              "attributes": {},
+                              "roles": [
+                                "data",
+                                "data_cold",
+                                "data_content",
+                                "data_frozen",
+                                "data_hot",
+                                "data_warm",
+                                "index",
+                                "ingest",
+                                "master",
+                                "ml",
+                                "remote_cluster_client",
+                                "search",
+                                "transform",
+                                "voting_only"
+                              ],
+                              "version": "%s"
+                            }
+                          },
+                          "transport_versions" : [
+                            {
+                              "node_id" : "nodeId1",
+                              "transport_version" : "%s"
+                            }
+                          ],
+                          "metadata": {
+                            "cluster_uuid": "clusterUUID",
+                            "cluster_uuid_committed": false,
+                            "cluster_coordination": {
+                              "term": 1,
+                              "last_committed_config": [
+                                "commitedConfigurationNodeId"
+                              ],
+                              "last_accepted_config": [
+                                "acceptedConfigurationNodeId"
+                              ],
+                              "voting_config_exclusions": [
+                                {
+                                  "node_id": "exlucdedNodeId",
+                                  "node_name": "excludedNodeName"
+                                }
+                              ]
+                            },
+                            "templates": {
+                              "template": {
+                                "order": 0,
+                                "index_patterns": [
+                                  "pattern1",
+                                  "pattern2"
+                                ],
+                                "settings": {
+                                  "index": {
+                                    "version": {
+                                      "created": "%s"
+                                    }
+                                  }
+                                },
+                                "mappings": {
+                                  "key1": {}
+                                },
+                                "aliases": {}
+                              }
+                            },
+                            "indices": {
+                              "index": {
+                                "version": 1,
+                                "mapping_version": 1,
+                                "settings_version": 1,
+                                "aliases_version": 1,
+                                "routing_num_shards": 1,
+                                "state": "open",
+                                "settings": {
+                                  "index": {
+                                    "number_of_shards": "1",
+                                    "number_of_replicas": "2",
+                                    "version": {
+                                      "created": "%s"
+                                    }
+                                  }
+                                },
+                                "mappings": {
+                                  "type": {
+                                    "type1": {
+                                      "key": "value"
+                                    }
+                                  }
+                                },
+                                "aliases": [
+                                  "alias"
+                                ],
+                                "primary_terms": {
+                                  "0": 1
+                                },
+                                "in_sync_allocations": {
+                                  "0": [
+                                    "allocationId"
+                                  ]
+                                },
+                                "rollover_info": {
+                                  "rolloveAlias": {
+                                    "met_conditions": {},
+                                    "time": 1
+                                  }
+                                },
+                                "system": false,
+                                "timestamp_range": {
+                                  "shards": []
+                                },
+                                "stats": {
+                                    "write_load": {
+                                      "loads": [-1.0],
+                                      "uptimes": [-1]
+                                    },
+                                    "avg_size": {
+                                        "total_size_in_bytes": 120,
+                                        "shard_count": 1
+                                    }
+                                },
+                                "write_load_forecast" : 8.0
+                              }
+                            },
+                            "index-graveyard": {
+                              "tombstones": []
+                            },
+                            "reserved_state" : { }
+                          },
+                          "routing_table": {
+                            "indices": {
+                              "index": {
+                                "shards": {
+                                  "0": [
+                                    {
+                                      "state": "STARTED",
+                                      "primary": true,
+                                      "node": "nodeId2",
+                                      "relocating_node": null,
+                                      "shard": 0,
+                                      "index": "index",
+                                      "allocation_id": {
+                                        "id": "%s"
+                                      },
+                                      "relocation_failure_info" : {
+                                        "failed_attempts" : 0
+                                      }
+                                    }
+                                  ]
+                                }
+                              }
+                            }
+                          },
+                          "routing_nodes": {
+                            "unassigned": [],
+                            "nodes": {
+                              "nodeId2": [
+                                {
+                                  "state": "STARTED",
+                                  "primary": true,
+                                  "node": "nodeId2",
+                                  "relocating_node": null,
+                                  "shard": 0,
+                                  "index": "index",
+                                  "allocation_id": {
+                                    "id": "%s"
+                                  },
+                                  "relocation_failure_info" : {
+                                    "failed_attempts" : 0
+                                  }
+                                }
+                              ],
+                              "nodeId1": []
+                            }
                           }
-                        }
-                      ],
-                      "nodeId1": []
-                    }
-                  }
-                }""", ephemeralId, Version.CURRENT, TransportVersion.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId)),
+                        }""",
+                    ephemeralId,
+                    Version.CURRENT,
+                    TransportVersion.CURRENT,
+                    Version.CURRENT.id,
+                    Version.CURRENT.id,
+                    allocationId,
+                    allocationId
+                )
+            ),
             Strings.toString(builder)
         );
 
@@ -391,175 +402,199 @@ public class ClusterStateTests extends ESTestCase {
         builder.endObject();
 
         assertEquals(
-            Strings.format("""
-                {
-                  "cluster_uuid" : "clusterUUID",
-                  "version" : 0,
-                  "state_uuid" : "stateUUID",
-                  "master_node" : "nodeId1",
-                  "blocks" : {
-                    "global" : {
-                      "1" : {
-                        "description" : "description",
-                        "retryable" : true,
-                        "disable_state_persistence" : true,
-                        "levels" : [
-                          "read",
-                          "write",
-                          "metadata_read",
-                          "metadata_write"
-                        ]
-                      }
-                    },
-                    "indices" : {
-                      "index" : {
-                        "2" : {
-                          "description" : "description2",
-                          "retryable" : false,
-                          "levels" : [
-                            "read",
-                            "write",
-                            "metadata_read",
-                            "metadata_write"
-                          ]
-                        }
-                      }
-                    }
-                  },
-                  "nodes" : {
-                    "nodeId1" : {
-                      "name" : "",
-                      "ephemeral_id" : "%s",
-                      "transport_address" : "127.0.0.1:111",
-                      "external_id" : "",
-                      "attributes" : { },
-                      "roles" : [
-                        "data",
-                        "data_cold",
-                        "data_content",
-                        "data_frozen",
-                        "data_hot",
-                        "data_warm",
-                        "index",
-                        "ingest",
-                        "master",
-                        "ml",
-                        "remote_cluster_client",
-                        "search",
-                        "transform",
-                        "voting_only"
-                      ],
-                      "version" : "%s"
-                    }
-                  },
-                  "transport_versions" : [
+            Strings.format(
+                """
                     {
-                      "node_id" : "nodeId1",
-                      "transport_version" : "%s"
-                    }
-                  ],
-                  "metadata" : {
-                    "cluster_uuid" : "clusterUUID",
-                    "cluster_uuid_committed" : false,
-                    "cluster_coordination" : {
-                      "term" : 1,
-                      "last_committed_config" : [
-                        "commitedConfigurationNodeId"
-                      ],
-                      "last_accepted_config" : [
-                        "acceptedConfigurationNodeId"
-                      ],
-                      "voting_config_exclusions" : [
-                        {
-                          "node_id" : "exlucdedNodeId",
-                          "node_name" : "excludedNodeName"
-                        }
-                      ]
-                    },
-                    "templates" : {
-                      "template" : {
-                        "order" : 0,
-                        "index_patterns" : [
-                          "pattern1",
-                          "pattern2"
-                        ],
-                        "settings" : {
-                          "index.version.created" : "%s"
+                      "cluster_uuid" : "clusterUUID",
+                      "version" : 0,
+                      "state_uuid" : "stateUUID",
+                      "master_node" : "nodeId1",
+                      "blocks" : {
+                        "global" : {
+                          "1" : {
+                            "description" : "description",
+                            "retryable" : true,
+                            "disable_state_persistence" : true,
+                            "levels" : [
+                              "read",
+                              "write",
+                              "metadata_read",
+                              "metadata_write"
+                            ]
+                          }
                         },
-                        "mappings" : {
-                          "key1" : { }
-                        },
-                        "aliases" : { }
-                      }
-                    },
-                    "indices" : {
-                      "index" : {
-                        "version" : 1,
-                        "mapping_version" : 1,
-                        "settings_version" : 1,
-                        "aliases_version" : 1,
-                        "routing_num_shards" : 1,
-                        "state" : "open",
-                        "settings" : {
-                          "index.number_of_replicas" : "2",
-                          "index.number_of_shards" : "1",
-                          "index.version.created" : "%s"
-                        },
-                        "mappings" : {
-                          "type" : {
-                            "type1" : {
-                              "key" : "value"
+                        "indices" : {
+                          "index" : {
+                            "2" : {
+                              "description" : "description2",
+                              "retryable" : false,
+                              "levels" : [
+                                "read",
+                                "write",
+                                "metadata_read",
+                                "metadata_write"
+                              ]
                             }
                           }
-                        },
-                        "aliases" : [
-                          "alias"
-                        ],
-                        "primary_terms" : {
-                          "0" : 1
-                        },
-                        "in_sync_allocations" : {
-                          "0" : [
-                            "allocationId"
+                        }
+                      },
+                      "nodes" : {
+                        "nodeId1" : {
+                          "name" : "",
+                          "ephemeral_id" : "%s",
+                          "transport_address" : "127.0.0.1:111",
+                          "external_id" : "",
+                          "attributes" : { },
+                          "roles" : [
+                            "data",
+                            "data_cold",
+                            "data_content",
+                            "data_frozen",
+                            "data_hot",
+                            "data_warm",
+                            "index",
+                            "ingest",
+                            "master",
+                            "ml",
+                            "remote_cluster_client",
+                            "search",
+                            "transform",
+                            "voting_only"
+                          ],
+                          "version" : "%s"
+                        }
+                      },
+                      "transport_versions" : [
+                        {
+                          "node_id" : "nodeId1",
+                          "transport_version" : "%s"
+                        }
+                      ],
+                      "metadata" : {
+                        "cluster_uuid" : "clusterUUID",
+                        "cluster_uuid_committed" : false,
+                        "cluster_coordination" : {
+                          "term" : 1,
+                          "last_committed_config" : [
+                            "commitedConfigurationNodeId"
+                          ],
+                          "last_accepted_config" : [
+                            "acceptedConfigurationNodeId"
+                          ],
+                          "voting_config_exclusions" : [
+                            {
+                              "node_id" : "exlucdedNodeId",
+                              "node_name" : "excludedNodeName"
+                            }
                           ]
                         },
-                        "rollover_info" : {
-                          "rolloveAlias" : {
-                            "met_conditions" : { },
-                            "time" : 1
-                          }
-                        },
-                        "system" : false,
-                        "timestamp_range" : {
-                          "shards" : [ ]
-                        },
-                        "stats" : {
-                          "write_load" : {
-                            "loads" : [
-                              -1.0
+                        "templates" : {
+                          "template" : {
+                            "order" : 0,
+                            "index_patterns" : [
+                              "pattern1",
+                              "pattern2"
                             ],
-                            "uptimes" : [
-                              -1
-                            ]
-                          },
-                          "avg_size" : {
-                            "total_size_in_bytes" : 120,
-                            "shard_count" : 1
+                            "settings" : {
+                              "index.version.created" : "%s"
+                            },
+                            "mappings" : {
+                              "key1" : { }
+                            },
+                            "aliases" : { }
                           }
                         },
-                        "write_load_forecast" : 8.0
-                      }
-                    },
-                    "index-graveyard" : {
-                      "tombstones" : [ ]
-                    },
-                    "reserved_state" : { }
-                  },
-                  "routing_table" : {
-                    "indices" : {
-                      "index" : {
-                        "shards" : {
-                          "0" : [
+                        "indices" : {
+                          "index" : {
+                            "version" : 1,
+                            "mapping_version" : 1,
+                            "settings_version" : 1,
+                            "aliases_version" : 1,
+                            "routing_num_shards" : 1,
+                            "state" : "open",
+                            "settings" : {
+                              "index.number_of_replicas" : "2",
+                              "index.number_of_shards" : "1",
+                              "index.version.created" : "%s"
+                            },
+                            "mappings" : {
+                              "type" : {
+                                "type1" : {
+                                  "key" : "value"
+                                }
+                              }
+                            },
+                            "aliases" : [
+                              "alias"
+                            ],
+                            "primary_terms" : {
+                              "0" : 1
+                            },
+                            "in_sync_allocations" : {
+                              "0" : [
+                                "allocationId"
+                              ]
+                            },
+                            "rollover_info" : {
+                              "rolloveAlias" : {
+                                "met_conditions" : { },
+                                "time" : 1
+                              }
+                            },
+                            "system" : false,
+                            "timestamp_range" : {
+                              "shards" : [ ]
+                            },
+                            "stats" : {
+                              "write_load" : {
+                                "loads" : [
+                                  -1.0
+                                ],
+                                "uptimes" : [
+                                  -1
+                                ]
+                              },
+                              "avg_size" : {
+                                "total_size_in_bytes" : 120,
+                                "shard_count" : 1
+                              }
+                            },
+                            "write_load_forecast" : 8.0
+                          }
+                        },
+                        "index-graveyard" : {
+                          "tombstones" : [ ]
+                        },
+                        "reserved_state" : { }
+                      },
+                      "routing_table" : {
+                        "indices" : {
+                          "index" : {
+                            "shards" : {
+                              "0" : [
+                                {
+                                  "state" : "STARTED",
+                                  "primary" : true,
+                                  "node" : "nodeId2",
+                                  "relocating_node" : null,
+                                  "shard" : 0,
+                                  "index" : "index",
+                                  "allocation_id" : {
+                                    "id" : "%s"
+                                  },
+                                  "relocation_failure_info" : {
+                                    "failed_attempts" : 0
+                                  }
+                                }
+                              ]
+                            }
+                          }
+                        }
+                      },
+                      "routing_nodes" : {
+                        "unassigned" : [ ],
+                        "nodes" : {
+                          "nodeId2" : [
                             {
                               "state" : "STARTED",
                               "primary" : true,
@@ -574,34 +609,19 @@ public class ClusterStateTests extends ESTestCase {
                                 "failed_attempts" : 0
                               }
                             }
-                          ]
+                          ],
+                          "nodeId1" : [ ]
                         }
                       }
-                    }
-                  },
-                  "routing_nodes" : {
-                    "unassigned" : [ ],
-                    "nodes" : {
-                      "nodeId2" : [
-                        {
-                          "state" : "STARTED",
-                          "primary" : true,
-                          "node" : "nodeId2",
-                          "relocating_node" : null,
-                          "shard" : 0,
-                          "index" : "index",
-                          "allocation_id" : {
-                            "id" : "%s"
-                          },
-                          "relocation_failure_info" : {
-                            "failed_attempts" : 0
-                          }
-                        }
-                      ],
-                      "nodeId1" : [ ]
-                    }
-                  }
-                }""", ephemeralId, Version.CURRENT, TransportVersion.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId),
+                    }""",
+                ephemeralId,
+                Version.CURRENT,
+                TransportVersion.CURRENT,
+                Version.CURRENT.id,
+                Version.CURRENT.id,
+                allocationId,
+                allocationId
+            ),
             Strings.toString(builder)
         );
 
@@ -629,181 +649,205 @@ public class ClusterStateTests extends ESTestCase {
         builder.endObject();
 
         assertEquals(
-            Strings.format("""
-                {
-                  "cluster_uuid" : "clusterUUID",
-                  "version" : 0,
-                  "state_uuid" : "stateUUID",
-                  "master_node" : "nodeId1",
-                  "blocks" : {
-                    "global" : {
-                      "1" : {
-                        "description" : "description",
-                        "retryable" : true,
-                        "disable_state_persistence" : true,
-                        "levels" : [
-                          "read",
-                          "write",
-                          "metadata_read",
-                          "metadata_write"
-                        ]
-                      }
-                    },
-                    "indices" : {
-                      "index" : {
-                        "2" : {
-                          "description" : "description2",
-                          "retryable" : false,
-                          "levels" : [
-                            "read",
-                            "write",
-                            "metadata_read",
-                            "metadata_write"
-                          ]
-                        }
-                      }
-                    }
-                  },
-                  "nodes" : {
-                    "nodeId1" : {
-                      "name" : "",
-                      "ephemeral_id" : "%s",
-                      "transport_address" : "127.0.0.1:111",
-                      "external_id" : "",
-                      "attributes" : { },
-                      "roles" : [
-                        "data",
-                        "data_cold",
-                        "data_content",
-                        "data_frozen",
-                        "data_hot",
-                        "data_warm",
-                        "index",
-                        "ingest",
-                        "master",
-                        "ml",
-                        "remote_cluster_client",
-                        "search",
-                        "transform",
-                        "voting_only"
-                      ],
-                      "version" : "%s"
-                    }
-                  },
-                  "transport_versions" : [
+            Strings.format(
+                """
                     {
-                      "node_id" : "nodeId1",
-                      "transport_version" : "%s"
-                    }
-                  ],
-                  "metadata" : {
-                    "cluster_uuid" : "clusterUUID",
-                    "cluster_uuid_committed" : false,
-                    "cluster_coordination" : {
-                      "term" : 1,
-                      "last_committed_config" : [
-                        "commitedConfigurationNodeId"
-                      ],
-                      "last_accepted_config" : [
-                        "acceptedConfigurationNodeId"
-                      ],
-                      "voting_config_exclusions" : [
-                        {
-                          "node_id" : "exlucdedNodeId",
-                          "node_name" : "excludedNodeName"
+                      "cluster_uuid" : "clusterUUID",
+                      "version" : 0,
+                      "state_uuid" : "stateUUID",
+                      "master_node" : "nodeId1",
+                      "blocks" : {
+                        "global" : {
+                          "1" : {
+                            "description" : "description",
+                            "retryable" : true,
+                            "disable_state_persistence" : true,
+                            "levels" : [
+                              "read",
+                              "write",
+                              "metadata_read",
+                              "metadata_write"
+                            ]
+                          }
+                        },
+                        "indices" : {
+                          "index" : {
+                            "2" : {
+                              "description" : "description2",
+                              "retryable" : false,
+                              "levels" : [
+                                "read",
+                                "write",
+                                "metadata_read",
+                                "metadata_write"
+                              ]
+                            }
+                          }
                         }
-                      ]
-                    },
-                    "templates" : {
-                      "template" : {
-                        "order" : 0,
-                        "index_patterns" : [
-                          "pattern1",
-                          "pattern2"
-                        ],
-                        "settings" : {
-                          "index" : {
-                            "version" : {
-                              "created" : "%s"
+                      },
+                      "nodes" : {
+                        "nodeId1" : {
+                          "name" : "",
+                          "ephemeral_id" : "%s",
+                          "transport_address" : "127.0.0.1:111",
+                          "external_id" : "",
+                          "attributes" : { },
+                          "roles" : [
+                            "data",
+                            "data_cold",
+                            "data_content",
+                            "data_frozen",
+                            "data_hot",
+                            "data_warm",
+                            "index",
+                            "ingest",
+                            "master",
+                            "ml",
+                            "remote_cluster_client",
+                            "search",
+                            "transform",
+                            "voting_only"
+                          ],
+                          "version" : "%s"
+                        }
+                      },
+                      "transport_versions" : [
+                        {
+                          "node_id" : "nodeId1",
+                          "transport_version" : "%s"
+                        }
+                      ],
+                      "metadata" : {
+                        "cluster_uuid" : "clusterUUID",
+                        "cluster_uuid_committed" : false,
+                        "cluster_coordination" : {
+                          "term" : 1,
+                          "last_committed_config" : [
+                            "commitedConfigurationNodeId"
+                          ],
+                          "last_accepted_config" : [
+                            "acceptedConfigurationNodeId"
+                          ],
+                          "voting_config_exclusions" : [
+                            {
+                              "node_id" : "exlucdedNodeId",
+                              "node_name" : "excludedNodeName"
                             }
-                          }
-                        },
-                        "mappings" : { },
-                        "aliases" : { }
-                      }
-                    },
-                    "indices" : {
-                      "index" : {
-                        "version" : 1,
-                        "mapping_version" : 1,
-                        "settings_version" : 1,
-                        "aliases_version" : 1,
-                        "routing_num_shards" : 1,
-                        "state" : "open",
-                        "settings" : {
-                          "index" : {
-                            "number_of_shards" : "1",
-                            "number_of_replicas" : "2",
-                            "version" : {
-                              "created" : "%s"
-                            }
-                          }
-                        },
-                        "mappings" : {
-                          "type" : {
-                            "type1" : {
-                              "key" : "value"
-                            }
-                          }
-                        },
-                        "aliases" : [
-                          "alias"
-                        ],
-                        "primary_terms" : {
-                          "0" : 1
-                        },
-                        "in_sync_allocations" : {
-                          "0" : [
-                            "allocationId"
                           ]
                         },
-                        "rollover_info" : {
-                          "rolloveAlias" : {
-                            "met_conditions" : { },
-                            "time" : 1
-                          }
-                        },
-                        "system" : false,
-                        "timestamp_range" : {
-                          "shards" : [ ]
-                        },
-                        "stats" : {
-                          "write_load" : {
-                            "loads" : [
-                              -1.0
+                        "templates" : {
+                          "template" : {
+                            "order" : 0,
+                            "index_patterns" : [
+                              "pattern1",
+                              "pattern2"
                             ],
-                            "uptimes" : [
-                              -1
-                            ]
-                          },
-                          "avg_size" : {
-                            "total_size_in_bytes" : 120,
-                            "shard_count" : 1
+                            "settings" : {
+                              "index" : {
+                                "version" : {
+                                  "created" : "%s"
+                                }
+                              }
+                            },
+                            "mappings" : { },
+                            "aliases" : { }
                           }
                         },
-                        "write_load_forecast" : 8.0
-                      }
-                    },
-                    "index-graveyard" : {
-                      "tombstones" : [ ]
-                    },
-                    "reserved_state" : { }
-                  },
-                  "routing_table" : {
-                    "indices" : {
-                      "index" : {
-                        "shards" : {
-                          "0" : [
+                        "indices" : {
+                          "index" : {
+                            "version" : 1,
+                            "mapping_version" : 1,
+                            "settings_version" : 1,
+                            "aliases_version" : 1,
+                            "routing_num_shards" : 1,
+                            "state" : "open",
+                            "settings" : {
+                              "index" : {
+                                "number_of_shards" : "1",
+                                "number_of_replicas" : "2",
+                                "version" : {
+                                  "created" : "%s"
+                                }
+                              }
+                            },
+                            "mappings" : {
+                              "type" : {
+                                "type1" : {
+                                  "key" : "value"
+                                }
+                              }
+                            },
+                            "aliases" : [
+                              "alias"
+                            ],
+                            "primary_terms" : {
+                              "0" : 1
+                            },
+                            "in_sync_allocations" : {
+                              "0" : [
+                                "allocationId"
+                              ]
+                            },
+                            "rollover_info" : {
+                              "rolloveAlias" : {
+                                "met_conditions" : { },
+                                "time" : 1
+                              }
+                            },
+                            "system" : false,
+                            "timestamp_range" : {
+                              "shards" : [ ]
+                            },
+                            "stats" : {
+                              "write_load" : {
+                                "loads" : [
+                                  -1.0
+                                ],
+                                "uptimes" : [
+                                  -1
+                                ]
+                              },
+                              "avg_size" : {
+                                "total_size_in_bytes" : 120,
+                                "shard_count" : 1
+                              }
+                            },
+                            "write_load_forecast" : 8.0
+                          }
+                        },
+                        "index-graveyard" : {
+                          "tombstones" : [ ]
+                        },
+                        "reserved_state" : { }
+                      },
+                      "routing_table" : {
+                        "indices" : {
+                          "index" : {
+                            "shards" : {
+                              "0" : [
+                                {
+                                  "state" : "STARTED",
+                                  "primary" : true,
+                                  "node" : "nodeId2",
+                                  "relocating_node" : null,
+                                  "shard" : 0,
+                                  "index" : "index",
+                                  "allocation_id" : {
+                                    "id" : "%s"
+                                  },
+                                  "relocation_failure_info" : {
+                                    "failed_attempts" : 0
+                                  }
+                                }
+                              ]
+                            }
+                          }
+                        }
+                      },
+                      "routing_nodes" : {
+                        "unassigned" : [ ],
+                        "nodes" : {
+                          "nodeId2" : [
                             {
                               "state" : "STARTED",
                               "primary" : true,
@@ -818,34 +862,19 @@ public class ClusterStateTests extends ESTestCase {
                                 "failed_attempts" : 0
                               }
                             }
-                          ]
+                          ],
+                          "nodeId1" : [ ]
                         }
                       }
-                    }
-                  },
-                  "routing_nodes" : {
-                    "unassigned" : [ ],
-                    "nodes" : {
-                      "nodeId2" : [
-                        {
-                          "state" : "STARTED",
-                          "primary" : true,
-                          "node" : "nodeId2",
-                          "relocating_node" : null,
-                          "shard" : 0,
-                          "index" : "index",
-                          "allocation_id" : {
-                            "id" : "%s"
-                          },
-                          "relocation_failure_info" : {
-                            "failed_attempts" : 0
-                          }
-                        }
-                      ],
-                      "nodeId1" : [ ]
-                    }
-                  }
-                }""", ephemeralId, Version.CURRENT, TransportVersion.CURRENT, Version.CURRENT.id, Version.CURRENT.id, allocationId, allocationId),
+                    }""",
+                ephemeralId,
+                Version.CURRENT,
+                TransportVersion.CURRENT,
+                Version.CURRENT.id,
+                Version.CURRENT.id,
+                allocationId,
+                allocationId
+            ),
             Strings.toString(builder)
         );
 

